descriptors/ModuleDescriptor-template.json

```
@@ -481,7 +497,9 @@
         "circulation-storage.cancellation-reasons.item.post",
         "circulation-storage.cancellation-reasons.item.put",
         "circulation-storage.cancellation-reasons.collection.delete",
-        "circulation-storage.cancellation-reasons.item.delete"
+        "circulation-storage.cancellation-reasons.item.delete",
+        "circulation-storage.patron-notice-policies.item.create",
+        "circulation-storage.patron-notice-policies.item.update"
```

This naming is inconsistent with the current convention of using put as the final postfix in the permission name for a PUT endpoint. Please can this be changed to fit with the current convention.
```suggestion
        "circulation-storage.patron-notice-policies.item.put"
```

---

changed to put